attendance/view.html

```diff
@@ -0,0 +1,197 @@
+---
+title: FIRST® Alumni &amp; Mentors Network at Michigan
+---
```

Comment:
I thought this was already PR'ed and merged?

Reply:
this was merged from `attendance-submit` into `dev`; larger features get merged into `dev` and then into `gh-pages`